custsignalmod.py

```diff
@@ -13,15 +23,16 @@
 OSC_INDICATORS = ['MACD', 'Stoch.RSI', 'Mom'] # Indicators to use in Oscillator analysis
 OSC_THRESHOLD = 2 # Must be less or equal to number of items in OSC_INDICATORS 
 MA_INDICATORS = ['EMA10', 'EMA20'] # Indicators to use in Moving averages analysis
-MA_THRESHOLD = 2 # Must be less or equal to number of items in MA_INDICATORS 
+MA_THRESHOLD = 2 # Must be less or equal to number of items in MA_INDICATORS
 INTERVAL = Interval.INTERVAL_5_MINUTES #Timeframe for analysis
 
 EXCHANGE = 'BINANCE'
 SCREENER = 'CRYPTO'
-PAIR_WITH = 'USDT'
-TICKERS = 'signalsample.txt'
-TIME_TO_WAIT = 4 # Minutes to wait between analysis
-FULL_LOG = False # List analysis result to console
+PAIR_WITH = parsed_config['trading_options']['PAIR_WITH']
+TICKERS = parsed_config['trading_options']['TICKERS_LIST']

```

Comment:
This needs to be a subset of the original list. These should not be duplicate becuase there are times where shitcoins are pumped and you will get a signal for a coin you dont want. Please create a new line item in the config file for tickers that are used in modules

Reply:
done, will be on next push - thank you!